stable/prometheus-to-sd/Chart.yaml

```diff
@@ -0,0 +1,4 @@
+apiVersion: v1
+description: A Helm chart for Kubernetes
```

Comment:
This should be the actual description of the chart. ;)

Reply:
Thanks for reviewing. Description updated.